contracts/BSCValidatorSet.sol

```diff
@@ -618,15 +666,15 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
       if (!isSameValidator(validatorSet[i], currentValidatorSet[i])) {
```

Comment:
Need change logic of `isSameValidator `, if voteAddr is not the same, then it is not the same.